msfrop

```diff
@@ -1,173 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
@dmaloney-r7, could we keep this file and have it call the gem? (so that you can still be in the msf directory and call it like: `./msrop` ?)